css/properties/cursor.json

```diff
@@ -359,7 +359,7 @@
                 "notes": "This cursor is only supported on macOS and Linux."
               },
               "firefox_android": {
-                "version_added": false
+                "version_added": "95"
```

Comment:
@makotokato This is the feature for `context-menu`. Above it says in chrome for android ` "notes": "This cursor is only supported on macOS and Linux."`

My _guess_ is that context menus are not supported on Android so actually all of the android platforms should be set to false, including this one for FF on Android. Do you happen to know?

Reply:
Firefox Android supports `context-menu` cursor type. Windows isn't still yet (https://bugzilla.mozilla.org/show_bug.cgi?id=258960)